spec/features/images_feature_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+feature 'images' do
+
+  context 'no images have been posted' do
+    scenario 'should display a prompt to post an image' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.